core-model-test/tests/src/test/resources/org/jboss/as/core/model/test/auditlog/standalone.xml

```diff
@@ -40,6 +41,7 @@
                <syslog-handler name="syslog-tls-default" formatter="json-two" >
                   <tls/>
                </syslog-handler>
+               <configuration-changes-handler name="in-memory" max-history="10"/>
```

Comment:
I think this should just be "in-memory-handler". The semantics should be the same as any other handler, except for the max-history behavior. It should record whatever the other handlers record, not limited to configuration changes, not disabled at boot, etc.